simplq/src/components/pages/Admin/ShareQueue.jsx

```diff
@@ -1,44 +1,23 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import FileCopyIcon from '@material-ui/icons/FileCopy';
-import { ShareButton } from '../../common/Button/Button.stories';
+import { useDispatch } from 'react-redux';
+import StandardButton from '../../common/Button';
+import { setInfoPopupMessage } from '../../../store/appSlice';
 
 const ShareQueue = (props) => {
-  const [clicked, setClicked] = useState(false);
   const shareUrl = `Hi! Use ${window.location.origin}/j/${props.queueName} to join my queue and get live updates.`;
-
-  useEffect(() => {
-    // https://www.alexhughes.dev/blog/setTimeout-with-hooks/
-    if (!clicked) {
-      return () => {};
-    }
-    // changes back to old text and color after 5 seconds
-    const timerId = setTimeout(() => setClicked(false), 5000);
-    return () => clearTimeout(timerId);
-  }, [clicked]);
+  const dispatch = useDispatch();
 
   const handleShareButtonClick = () => {
-    setClicked(true);
+    dispatch(setInfoPopupMessage('Copied to clipboard'));
   };
 
-  const CopyToClipboardButton = () =>
-    clicked ? (
-      <>
-        <img src="/images/verified-24px.svg" alt="copied" />
-        <p>Copied to clipboard</p>
-      </>
-    ) : (
-      <>
-        <FileCopyIcon />
-        <p>Copy to clipboard</p>
-      </>
-    );
-
   return (
     <CopyToClipboard text={shareUrl}>
-      <ShareButton onClick={handleShareButtonClick} outlined={!clicked}>
-        <CopyToClipboardButton />
-      </ShareButton>
+      <StandardButton onClick={handleShareButtonClick} icon={<FileCopyIcon />}>
+        Copy to clipboard
+      </StandardButton>
```

Comment:
Copy Queue Link makes sense. I think we should keep "Share Queue" for a different use case where we can detect that the browser is mobile, and we put a "Share" button if there is support for it. (It will then open the app to share via, Whatsapp, FB etc). I have not checked if this is possible, though.